submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -53,9 +53,11 @@ class SubmissionService(
         )
     }
 
+    @Suppress("MagicNumber")
     @RabbitListener(queues = [SUBMISSION_REQUEST_QUEUE], concurrency = "1-1")
     fun processSubmission(request: SubmissionRequestMessage) {
         logger.info { "received process message for submission ${request.submission}" }
+        Thread.sleep(30_000L)
```

Comment:
I understand this is for testing purposes. I think we should add a TODO so we don't forget to remove it or even removing it before merging now that it's tested.